lib/src/json_schema/validator.dart

```diff
@@ -258,7 +288,7 @@ class Validator {
     } else {
       final items = schema.itemsList;
 
-      if (items != null) {
+      if (items != null && _vocabulary.contains(SupportedVocabularies.APPLICATOR)) {
```

Comment:
@michaelcarter-wf I chatted with @robbielamb-wf about what we should do in the case where the applicator vocabulary is not used, so things like `items` are not keywords. We decided it probably does not make sense to proceed with the usual per-item validation if `items` itself is not a keyword.